akka-testkit/src/main/scala/akka/testkit/ExplicitlyTriggeredScheduler.scala

```diff
@@ -38,12 +38,12 @@ class ExplicitlyTriggeredScheduler(config: Config, log: LoggingAdapter, tf: Thre
     schedule(delay, None, runnable)
 
   /**
-    * Advance the clock by the specified duration.
-    *
-    * We will not add a dilation factor to this amount, since the scheduler API also does not apply dilation.
-    * If you want the amount of time passed to be dilated, apply the dilation before passing the delay to
-    * this method.
-    */
+   * Advance the clock by the specified duration.
+   *
+   * We will not add a dilation factor to this amount, since the scheduler API also does not apply dilation.
+   * If you want the amount of time passed to be dilated, apply the dilation before passing the delay to
+   * this method.
+   */
```

Comment:
"This will execute all outstanding jobs on the calling thread before returning" or such?

Reply:
👍. (It will also execute any jobs that are scheduled to be executed within the specified period by those outstanding jobs, but that might be a subtlety that doesn't really belong in the scaladoc here)